.ci_support/azure-linux-64-comp7.yaml

```diff
@@ -8,3 +8,29 @@ channel_sources:
   - conda-forge,defaults
 docker_image:
   - condaforge/linux-anvil-comp7
```

Comment:
Can you remove this spurious edit?

Reply:
Oops I restored original version. Sorry for that.